cmake/project-config.cmake.in

```diff
@@ -6,9 +6,7 @@ include(CMakeFindDependencyMacro)
 list(APPEND CMAKE_MODULE_PATH "${PACKAGE_PREFIX_DIR}/@PROXYFMU_CMAKE_INSTALL_DIR@")
 
 find_dependency(Boost COMPONENTS filesystem REQUIRED)
-
-find_dependency(FMILIB REQUIRED)
-find_dependency(THRIFT REQUIRED)
-find_dependency(CLI11 REQUIRED)
```

Comment:
Should this have been removed?

Reply:
It looks to me like CLI11 is only used by the `proxyfmu` executable, and not by the library, and if so, it should not be a dependency of anyone who uses the library.